src/main/webapp/app/exercise/result-history/result-history.component.ts

```diff
@@ -22,6 +23,8 @@ export class ResultHistoryComponent implements OnChanges {
     readonly MissingResultInfo = MissingResultInformation;
 
     results = input.required<Result[]>();
+    participationInput = input<Participation>();
```

Comment:
This is quite odd to read, especially when using the component, can we rename it?

`participationInput` -> `participation`

`participation` -> `displayedParticipation`


Signal migration optional for my approve: 
I do also think that we can easily change `displayedResults` to a signal and make `displayedParticipation` a computed signal instead, am I missing something?